hw02/PlayerController.cs

```diff
@@ -15,17 +18,36 @@ 1. Player or Character's Name
                 2. Movement Speed as a float
                 3. Gravity Value as a float
                 4. Jump Speed as a float
-    
+    */
+
+    private string playerName = "JC"; // Player Name
+    private float movementSpeed = 5.5; // Movement Speed
+    private float gravityValue = 9.8; //Gravity Value
+    private float jumpSpeed = 3.5; //Jump Speed
+
+    /*
     When the game starts, we would like to find our character. In our Unity Editor, we have a 3D model of our player and the player is 
     represented as a component in Unity. How do we grab our player controller in code and where should we write this line of code in this
     C# document (PlayerController.cs)?
     */
-    
+    public PlayerController player;
+
     // Start is called before the first frame update
     void Start()
     {
+        // Print the variables to the Unity console when the game starts.
+        Debug.Log("Player Name: " + playerName);
+        Debug.Log("Movement Speed: " + movementSpeed);
+        Debug.Log("Gravity Value: " + gravityValue);
+        Debug.Log("Jump Speed: " + jumpSpeed);
+
+        player = this.GetComponent<PlayerController>();
         // TODO: Problem 3: Describe when you would put code in the Start() function instead of the Update() function. 
         // What is the difference between Start() and Update()?
+        /*
+        Use the Start() function to initialize variables, set up references, and configure the initial game state. It is called once when the script is first executed
```

Comment:
That's correct! There is another function similar to start(), it's called Awake. The awake() will be called when an object first become enabled and active.